### PR TITLE
Fixes Energy Harvester NTos app

### PR DESCRIPTION
**Pull request by ghost**
### Intent of your Pull Request

The previous energy harvester PR was 90% complete and was merged in that state. The 10% incomplete bits were the NTOS app which isn't really that important since it's basically just a backup, this PR finishes it.

#### Changelog

:cl:  
tweak: Fixes the energy harvester ntos app
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
GXQZ6gxfp1
3MrHH3HXT8

Second pic has working at green instead of red in final prod build

File: code/modules/modular_computers/file_system/programs/energyharvestercontrol.dm [hunk 1]
```diff
@@ -11,20 +11,13 @@
 	size = 1
 	tgui_id = "NtosEnergyHarvesterController"
 	ui_x = 300
-	ui_y = 350
+	ui_y = 420
 	var/obj/item/energy_harvester/moneysink
 
 ///updates icon
 /datum/computer_file/program/energy_harvester_control/process_tick()
```
Comment (Bibby0110) [inline code/modules/modular_computers/file_system/programs/energyharvestercontrol.dm:19]: this doesn't actually no anything, remove

Comment (ghost) [reply code/modules/modular_computers/file_system/programs/energyharvestercontrol.dm:19]: Yes it does you silly gamer.

Comment (ghost) [reply code/modules/modular_computers/file_system/programs/energyharvestercontrol.dm:19]: Waiting on sproters to hand me the sprotes so I can update process_tick()

Comment (ghost) [reply code/modules/modular_computers/file_system/programs/energyharvestercontrol.dm:19]: sprotes are in

**Review by fluffe9911 — APPROVED** [review]
ok

**Pull request merged**